core/src/network/handler/mod.rs

```diff
@@ -9,6 +9,30 @@ use crate::{
 use holochain_net_connection::{net_connection::NetHandler, protocol_wrapper::ProtocolWrapper};
 use std::{convert::TryFrom, sync::Arc};
 
+// FIXME: Temporary hack to ignore messages incorrectly sent to us by the networking
+// module that aren't really meant for us:
+fn is_me(c: &Arc<Context>, dna_hash: &str, agent_id: &str) -> bool {
```

Comment:
@zippy finding the name 'is_me' is a little confusing... what "is me"? what "is us"? as per the comment. Who is me, who is us?

Reply:
Should/could this be called "message_is_for_me"?